From a56d9f6eb9e02062666e8f374d6a9325fb09d29f Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Wed, 21 Oct 2020 15:31:08 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568: correct gic address

There is no GICC/GICH/GICV memory-mapped address for RK3568 SoC.

Change-Id: Ie128cc2e5994b703d1fba9df44216331ed3c6c54
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index 2be4ccde907c..4972db811dbe 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -113,15 +113,12 @@
 		interrupt-controller;
 
 		reg = <0x0 0xfd400000 0 0x10000>, /* GICD */
-		      <0x0 0xfd460000 0 0xc0000>, /* GICR */
-		      <0x0 0xfd800000 0 0x10000>, /* GICC */
-		      <0x0 0xfd810000 0 0x10000>, /* GICH */
-		      <0x0 0xfd820000 0 0x10000>; /* GICV */
+		      <0x0 0xfd460000 0 0xc0000>; /* GICR */
 		interrupts = <GIC_PPI 9 IRQ_TYPE_LEVEL_HIGH>;
-		its: interrupt-controller@fd420000 {
+		its: interrupt-controller@fd440000 {
 			compatible = "arm,gic-v3-its";
 			msi-controller;
-			reg = <0x0 0xfd420000 0x0 0x20000>;
+			reg = <0x0 0xfd440000 0x0 0x20000>;
 		};
 	};
 
-- 
2.35.3

